formula/src/main/java/com/instacart/formula/FormulaContext.kt

```diff
@@ -102,6 +102,18 @@ abstract class FormulaContext<out Input, State> internal constructor(
         input: ChildInput
     ): ChildOutput
 
+    /**
+     * Returns the latest output of the [child] formula or null if the child formula has encountered
+     * an exception. Child formulas that encounter exceptions will be be terminated and will not
+     * be run again. Formula runtime ensures the [child] is running, manages its internal state
+     * and will trigger `evaluate` if needed.
+     */
+    abstract fun <ChildInput, ChildOutput> child(
```

Comment:
Use separate method in FormulaContext to avoid nullability concerns complicating API contract of FormulaManager

Reply:
There is also a method without an `input` property for `IFormula<Unit, T>`, we should similarly add `onError` support.